src/store/modules/user.store.js

```diff
@@ -14,8 +15,40 @@ export const userStore = {
                     users => commit('getAllSuccess', users),
                     error => commit('getAllFailure', error)
                 );
-        }
+        },
+        setting({ dispatch, commit }, {
+            profilePicture,
+            username,
+            bio,
+            password1,
+            password2,
+        }) {
+            commit('settingRequest', { username });
+
+            userService.setting(profilePicture, username, bio, password1, password2)
+            .then(
+                user => {
+                    // console.log(user);
+                    let storedProfile = JSON.parse(localStorage.getItem('user'));
+                    console.log(storedProfile.profile.user);
+                    storedProfile.profile = user.profile;
+                    storedProfile.profile = user.updateFields;
+                    localStorage.setItem('user', JSON.stringify(storedProfile));
+                    console.log(storedProfile.updateFields);
+                    console.log(user);
+
+                    commit('loginSuccess', user);
```

Comment:
There is no requirement of this. Please remove anything you feel have blindly copied and has no use for updation.